### PR TITLE
HDDS-2726. start/StartFromDockerHub.md translated to Chinese

### PR DESCRIPTION
**Pull request by iamabug**
## What changes were proposed in this pull request?

two .md files modifed

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-2726

## How was this patch tested?

hugo server -D

### CODE REVIEW

**Comment by iamabug** [comment]
FYI, commit message is mistyped, it should be HDDS-2726

File: hadoop-hdds/docs/content/start/StartFromDockerHub.zh.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Simple Single Ozone
+title: 简易 Ozone
```
Comment (timmylicheng) [inline hadoop-hdds/docs/content/start/StartFromDockerHub.zh.md]: how about 'Single'? And from the context, what does Ozone mean here, like ozone server or ozone cluster?

File: hadoop-hdds/docs/content/start/StartFromDockerHub.zh.md [hunk 2]
```diff
@@ -21,91 +21,76 @@ weight: 10
...
 
 {{< requirements >}}
- * Working docker setup
- * AWS CLI (optional)
+ * docker
+ * AWS CLI（可选）
 {{< /requirements >}}
 
-# Ozone in a Single Container
+# 单容器 Ozone
```
Comment (timmylicheng) [inline hadoop-hdds/docs/content/start/StartFromDockerHub.zh.md]: I think this is supposed to align with the 'all-in-one' terminology. Some place translates this to 全家桶.

File: hadoop-hdds/docs/content/start/StartFromDockerHub.zh.md [hunk 3]
```diff
@@ -21,91 +21,76 @@ weight: 10
...
-# Running S3 Clients
+# 运行 S3 客户端
 
-Once the cluster is booted up and ready, you can verify its status by
-connecting to the SCM's UI at [http://localhost:9876](http://localhost:9876).
+集群启动就绪后，你可以连接 SCM 的 UI 来验证它的状态，地址为（[http://localhost:9876](http://localhost:9876)）。
 
-The S3 gateway endpoint will be exposed at port 9878. You can use Ozone's S3
-support as if you are working against the real S3.
+S3 网关的端口为 9878，如果你在用 S3 工作，可以考虑使用 Ozone 的 S3 功能。
```
Comment (timmylicheng) [inline hadoop-hdds/docs/content/start/StartFromDockerHub.zh.md]: 如果你的系统正在用S3作为存出方案...

File: hadoop-hdds/docs/content/start/StartFromDockerHub.zh.md (hunk 2 above)
Comment (iamabug) [reply hadoop-hdds/docs/content/start/StartFromDockerHub.zh.md]: Sounds good, will do

File: hadoop-hdds/docs/content/start/StartFromDockerHub.zh.md (hunk 1 above)
Comment (iamabug) [reply hadoop-hdds/docs/content/start/StartFromDockerHub.zh.md]: 1. 'Single' is kind of implied by 'Simple'
2. docker-compose way is also introduced in this page, which is not so 'Single'
3. I cannot think of a good way to translate 'Single'

As for what Ozone means, I am not sure if we should emphasize that it is a cluster or server here, it kind of feels like "Ozone is Ozone".

File: hadoop-hdds/docs/content/start/StartFromDockerHub.zh.md (hunk 3 above)
Comment (iamabug) [reply hadoop-hdds/docs/content/start/StartFromDockerHub.zh.md]: Agreed, much better.

Comment (xiaoyuyao) [reply hadoop-hdds/docs/content/start/StartFromDockerHub.zh.md]: 用S3作为存出方案=>使用基于AWS S3协议的存储方案

File: hadoop-hdds/docs/content/start/StartFromDockerHub.zh.md [hunk 4]
```diff
@@ -21,91 +21,76 @@ weight: 10
...
 
 {{< requirements >}}
- * Working docker setup
- * AWS CLI (optional)
+ * docker
+ * AWS CLI（可选）
 {{< /requirements >}}
 
-# Ozone in a Single Container
+# 单容器全家桶 Ozone
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/start/StartFromDockerHub.zh.md]: 所以Ozone服务在单个容器

File: hadoop-hdds/docs/content/start/StartFromDockerHub.zh.md [hunk 5]
```diff
@@ -21,91 +21,76 @@ weight: 10
...
-This command will pull down the ozone image from docker hub and start all
-ozone services in a single container. <br>
-This container will run the required metadata servers (Ozone Manager, Storage
-Container Manager) one data node  and the S3 compatible REST server
-(S3 Gateway).
+这个命令会从 Docker Hub 拉取 ozone 镜像并在一个容器中启动所有 ozone 服务，包括必要的元数据服务（Ozone Manager，Storage Container Manager）、一个数据节点和兼容 S3
+ 的 REST 服务（S3 网关）。
 
-# Local multi-container cluster
+# 本地多容器集群
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/start/StartFromDockerHub.zh.md]: Ozone服务在多个独立的容器

File: hadoop-hdds/docs/content/start/StartFromDockerHub.zh.md [hunk 6]
```diff
@@ -21,91 +21,76 @@ weight: 10
...
 
-If you would like to use a more realistic pseudo-cluster where each components
-run in own containers, you can start it with a docker-compose file.
+如果你想要一个更符合实际的伪集群，以便让各个组件运行在单独的容器中，可以使用 docker-compose 配置文件来启动。
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/start/StartFromDockerHub.zh.md]: 如果你需要一个更类似生产环境的集群，Ozone发行包自带的docker-compose可以让Ozone服务组件在各自独立的容器中运行。

File: hadoop-hdds/docs/content/start/StartFromDockerHub.zh.md [hunk 7]
```diff
@@ -21,91 +21,76 @@ weight: 10
...
-Here is how you create buckets from command line:
+从命令行创建桶的命令为：
 
 ```bash
 aws s3api --endpoint http://localhost:9878/ create-bucket --bucket=bucket1
 ```
 
-Only notable difference in the above command line is the fact that you have
-to tell the _endpoint_ address to the aws s3api command.
+唯一的区别在于你需要把 endpoint 的地址告诉 aws s3api 命令。
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/start/StartFromDockerHub.zh.md]: 唯一的区别在于你需要在运行aws s3api 命令行的时候用 --endpoint选项指定ozone S3 网关的地址。

File: hadoop-hdds/docs/content/start/StartFromDockerHub.zh.md (hunk 4 above)
Comment (qiusuo106) [reply hadoop-hdds/docs/content/start/StartFromDockerHub.zh.md]: 是“所有 Ozone 服务在单个容器”吗？

File: hadoop-hdds/docs/content/start/StartFromDockerHub.zh.md [hunk 8]
```diff
@@ -21,91 +21,76 @@ weight: 10
...
+注意：对于单容器 ozone 来说，REDUCED_REDUNDANCY 参数是必需的，因为它只有一个数据节点。</div>
+我们可以对桶运行 list 命令来验证文件是否上传成功：
 
 ```bash
 aws s3 --endpoint http://localhost:9878 ls s3://bucket1/testfile
 ```
 
-<div class="alert alert-info" role="alert"> You can also check the internal
-bucket browser supported by Ozone S3 interface by clicking on the below link.
+<div class="alert alert-info" role="alert"> 你也可以点击下面的链接，查看 Ozone 的 S3 接口内置的桶浏览器。
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/start/StartFromDockerHub.zh.md]: 的 S3 接口内置的 =》 S3 网关自带的浏览器去查看桶内的文件。

**Comment by xiaoyuyao** [comment]
LGTM overall， some comments inline

**Comment by iamabug** [comment]
new commit is pushed. @xiaoyuyao

**Comment by iamabug** [comment]
@xiaoyuyao how about this patch ?

**Comment by xiaoyuyao** [comment]
LGTM, +1. I will merge it shortly.

**Pull request merged**